lib/css/res.scss

```diff
@@ -1836,6 +1836,21 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 			content: attr(aftercontent);
 		}
 	}
+
+	&.selected a {
+		&:first-child {
+			border-right: none;
+		}
+
+		&:last-child {
+			border-left: none;
+		}
+
+		&:only-child {
```

Comment:
If you modify `&.selected a {` to `&.selected a:not(:only-child) {` then you can remove this block.